pkg/storage/localstore/export_test.go

```diff
@@ -67,14 +68,14 @@ func TestExportImport(t *testing.T) {
 	}
 
 	for a, want := range chunks {
-		addr := chunk.Address([]byte(a))
-		ch, err := db2.Get(context.Background(), chunk.ModeGetRequest, addr)
+		addr := swarm.MustParseHexAddress(a)
+		ch, err := db2.Get(context.Background(), storage.ModeGetRequest, addr)
 		if err != nil {
 			t.Fatal(err)
 		}
 		got := ch.Data()
 		if !bytes.Equal(got, want) {
-			t.Fatalf("chunk %s: got data %x, want %x", addr.Hex(), got, want)
+			t.Fatalf("chunk %s: got data %s, want %x", addr, got, want)
```

Comment:
shouldn't the last `%x` be also `%s`?

Reply:
Actually, this change is not correct, the second word should stay the same.